.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## "Ready for Review" checklist
+
+- [ ] The Vercel preview link has been added to this PR's description
+- [ ] The Asana task has been added to this PR's description
+- [ ] This PR's link has been added to the "📐 Pull Request" field on the Asana task
+- [ ] This Vercel preview link has been added to the "🔍 Preview" field on the Asana task
+- [ ] The description template has been filled in below
+
+---
+
+## Relevant links
+
+- [Preview link](url) 🔎
+- [Asana task](url) 🎟️
+
+## What
+
+_Briefly list out the changes proposed in this PR._
```

Comment:
What do you think about making the section descriptions comments? That way we don't need to delete them as we're filling out the PR. Just one less point of friction!

Reply:
Great idea! I'll make this change